architecture/index.html

```diff
@@ -174,7 +174,7 @@ <h2>
       A (well-chosen) URL <em>combines</em><br>
       identification and location.
     </h2>
-    <p><a href="http://en.wikipedia.org/wiki/Marissa_Mayer">http://en.wikipedia.org/wiki/Marissa_Mayer</a></p>
+    <p><a href="https://en.wikipedia.org/wiki/Marissa_Mayer">https://en.wikipedia.org/wiki/Marissa_Mayer</a></p>
```

Comment:
Let's keep these at HTTP, such that the example is still simple (port 80, plaintext etc.)

Reply:
Should this also be the case for line 275, 321 and 518? (wikipedia too)

Reply:
Yes please, given that it's where we walk through the basics of HTTP (https://rubenverborgh.github.io/WebFundamentals/architecture/#example-request).

Reply:
Good point